test/CMakeLists.txt

```diff
@@ -54,9 +56,12 @@ target_link_libraries(linux_features_aggregator_test all_libraries)
 add_test(NAME linux_features_aggregator_test COMMAND linux_features_aggregator_test)
 ##------------------------------------------------------------------------------
 ## cpuinfo_x86_test
+if(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
 add_executable(cpuinfo_x86_test cpuinfo_x86_test.cc ../src/cpuinfo_x86.c)
+target_compile_definitions(stack_line_reader PUBLIC CPU_FEATURES_MOCK_CPUID_X86)
```

Comment:
good target ? if yes you should separate it from `cpuinfo_x86_test` stuff

Reply:
Good catch